pkg/agent/nplagent/k8s/annotations.go

```diff
@@ -0,0 +1,164 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8s
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/vmware-tanzu/antrea/pkg/util/env"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+const NPLAnnotationStr = "npl.antrea.io"
+
+type NPLEPAnnotation struct {
+	PodPort  string `json:"Podport"`
+	NodeIP   string `json:"Nodeip"`
+	NodePort string `json:"Nodeport"`
```

Comment:
The json keys are normally in this style: podPort, nodeIP, nodePort.
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L61

Reply:
I'll update it.